auslib/test/admin/views/test_releases.py

```diff
@@ -1016,18 +1017,308 @@ def testReleasesPost(self):
 """))
 
 
-def byteify(input):
-    if isinstance(input, dict):
-        res = {}
-        for key, value in input.items():
-            res[byteify(key)] = byteify(value)
-        return res
-    elif isinstance(input, list):
-        return [byteify(element) for element in input]
-    elif isinstance(input, unicode):
-        return input.encode('utf-8')
-    else:
-        return input
+class TestReleasesScheduledChanges(ViewTest):
+    maxDiff = 10000
+
+    def setUp(self):
+        super(TestReleasesScheduledChanges, self).setUp()
+        dbo.releases.scheduled_changes.t.insert().execute(
+            sc_id=1, scheduled_by="bill", change_type="insert", data_version=1, base_name="m", base_product="m",
+            base_data=createBlob(dict(name="m", hashFunction="sha512", schema_version=1))
+        )
+        dbo.releases.scheduled_changes.history.t.insert().execute(change_id=1, changed_by="bill", timestamp=50, sc_id=1)
+        dbo.releases.scheduled_changes.history.t.insert().execute(
+            change_id=2, changed_by="bill", timestamp=51, sc_id=1, scheduled_by="bill", change_type="insert", data_version=1, base_name="m",
+            base_product="m", base_data=createBlob(dict(name="m", hashFunction="sha512", schema_version=1))
+        )
+        dbo.releases.scheduled_changes.signoffs.t.insert().execute(sc_id=1, username="bill", role="releng")
```

Comment:
Yeah, I sortof struggled to decide what was the best balance here. We're basically just testing base class stuff from auslib/admin/views/scheduled_changes.py and history.py here, so it's a bit of a tough sell to add the same tests for each type of Scheduled Change. OTOH, adding these tests found bugs in the base classes with handling of JSON columns.